### PR TITLE
[ Besu ] Enablement of DNS support in HL Besu

### PR DESCRIPTION
**Pull request by ragalz**
**Changelog**
- Update deployment to allow DNS on network.yaml

 

**Reviewed by**
@developer_github_id

 

**Linked issue**
#983

### CODE REVIEW

File: platforms/hyperledger-fabric/releases/dev/supplychain/orderer/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (sownak) [inline platforms/hyperledger-fabric/releases/dev/supplychain/orderer/namespace.yaml:1]: releases folder should not be checked-in

File: platforms/shared/charts/ambassador/templates/service.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ metadata:
   namespace: default
   {{- if .Values.ambassador.eip }}
   annotations:
-    service.beta.kubernetes.io/aws-load-balancer-type: nlb
+    service.beta.kubernetes.io/aws-load-balancer-type: elb
```
Comment (sownak) [inline platforms/shared/charts/ambassador/templates/service.yaml:25]: this should not be changed as it impacts Indy

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by jagpreetsinghsasan — COMMENTED** [review]
Please do not delete the releases/README.md file

**Review by suvajit-sarkar — APPROVED** [review]
incorporate the changes requested by @jagpreetsinghsasan 
reset LGTM

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Pull request merged**